packages/terra-clinical-result/src/proptypes/interpretationPropTypes.js

```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types';
+
+/**
+ *  Enum for possible Result Interpretation values (also called `Clinical Severity` and `Normalcy`)
+ */
+const interpretationPropOneOf = PropTypes.oneOf([
+  'CRITICAL',
+  'EXTREMEHIGH',
+  'EXTREMELOW',
+  'PANICHIGH',
+  'PANICLOW',
+  'VABNORMAL',
+  'POSITIVE',
+  'ABNORMAL',
+  'HIGH',
+  'LOW',
+  'NORMAL',
+  'NEUTRAL',
+]);
+
+export default interpretationPropOneOf;
+export {
+  interpretationPropOneOf,
```

Comment:
Why do we need to export this twice?

Reply:
Removed extraneous export https://github.com/cerner/terra-clinical/pull/574/commits/9eeacc2469da67d5f2d9ad3ce7cc533a407dbf13?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&file-filters%5B%5D=.scss&file-filters%5B%5D=.snap#diff-ed909a03ef446a8f025880c76d077aa5R21